importer/api/__init__.py

```diff
@@ -92,7 +92,8 @@ def import_course_from_path(path, repo_id, user_id):
     Returns:
         course (learningresources.Course)
     """
-    bundle = XBundle()
+    bundle = XBundle(
+        keep_urls=True, keep_studio_urls=True, preserve_url_name=True)
```

Comment:
nit: prefer end paren to be on new line if newline character is first character after the opening paren